### ISSUE TITLE
Guild Subdomains

### ISSUE THREAD

**Issue by jonzenor** [jonzenor/ChristianGuilds#221]
Look here for some examples: 
* https://likegeeks.com/dynamic-laravel-subdomain-routing/
* https://dzone.com/articles/how-to-setup-specific-subdomain-routing-in-laravel
* https://dev.to/arielmejiadev/set-dynamic-subdomains-in-laravel-forge-5fi8
* https://stackoverflow.com/questions/24327868/php-laravel-allow-users-to-use-a-custom-domain-name

- Guild Subdomains would be a paid feature. Difference between subdomain and full domain?

**Comment by Lectric1** [comment]
The difference between the two is subdomains are customdomain.maincompanysite.com whereas a full domain is customdomain.com

**Comment by Lectric1** [comment]
either you would have to purchase domain names for the normal full domains, or you would have them bring it. also note that you can have unlimited free subdomains with your domain.

**Comment by jonzenor** [comment]
Yeah... I know what subdomains are... :D I meant from the application perspective. I have to configure my server differently to dynamically allow for domains to be used, and then have to reconfigure how all of the routing works based on if you are using a standard slug, a subdomain, or a custom domain. Getting those options to all work together may be a big pain, and I will need to probably limit what options I allow. So thought must be given.
